operator/flags.go

```diff
@@ -283,14 +257,6 @@ func init() {
 	flags.Bool(option.Version, false, "Print version information")
 	option.BindEnv(option.Version)
 
-	// Deprecated, remove in 1.9
-	flags.Uint16Var(&apiServerPort, "api-server-port", 9234, "Port on which the operator should serve API requests")
-	flags.MarkDeprecated("api-server-port", fmt.Sprintf("Please use %s instead", operatorOption.OperatorAPIServeAddr))
-
-	// Deprecated, remove in 1.9
-	flags.StringVar(&operatorMetrics.Address, "metrics-address", ":6942", "Address to serve Prometheus metrics")
```

Comment:
This seems to be the only place where `operator/metrics.Address` is set, so it could be removed as well (and `operator/metrics.getPrometheusServerAddr()` could be replaced by `operatorOption.Config.OperatorPrometheusServeAddr` as well)